src/terraform/examples/HPC Cache/vdbench/nfsfiler/main.tf

```diff
@@ -7,18 +7,27 @@ locals {
   network_resource_group_name   = "vdbench_network_rg"
   vmss_resource_group_name      = "vdbench_vmss_rg"
 
-  // HPC Cache Throughput SKU - 3 allowed values for throughput (GB/s) of the cache
+  // HPC Cache Throughput SKU - allowed values for throughput (GB/s) of the cache
   //  Standard_2G
   //  Standard_4G
   //  Standard_8G
+  //  Standard_L4_5G 
+  //  Standard_L9G 
+  //  Standard_L16G 
+  //  Premium_5G
```

Comment:
Should we remove Premium Skus and sizes from this file and azureblobfiler/main.tf

Reply:
Removed